tests/ypd/model/test_prototype_server.py

```diff
@@ -0,0 +1,38 @@
+import os
+import tempfile
+from unittest import TestCase
+from unittest.mock import patch
+
+import pytest
+
+from ypd.model import Base, Session, engine, project
+
+
+class TestProject(TestCase):
```

Comment:
You should name this something like TestServer
Also prototype_server doesn't exist anymore. The file should be named test_server.py